src/PluralResolver.js

```diff
@@ -102,6 +102,7 @@ class PluralResolver {
     }
 
     this.rules = createRules();
+    this.pluralRulesCache = new Map();
```

Comment:
Can you add a comment about why we created this cache? Mention the performance costs in React Native and the trade-offs